### PR TITLE
Declarative test title

### PR DESCRIPTION
**Pull request by tfredett**

### CODE REVIEW

File: test/Multichain.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ describe('Multichain', () => {
             })
         })
         describe('StreamItemsByKey', () => {
-            it('should return stream items by key', function(done) {
+            it('Returns a stream items by key', function(done) {
```
Comment (tfredett) [inline test/Multichain.js]: Note: should read `Returns a stream of items by key`

File: test/PyShell.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ let PyShellOptions = PyShellLib.PyShellOptions
 let PyShellMode = PyShellLib.Mode
 
 describe('PyShellOptions', () => {
-    it('is created sucessfully', () => {
+    it('Is created sucessfully', () => {
```
Comment (tfredett) [inline test/PyShell.js]: Note: typo, should be `successfully`

File: test/Secure.js [hunk 3]
```diff
@@ -164,58 +163,55 @@ describe('HDKey', () => {
...
+    var dh = new Diffie(strength).Serialize()
     var pubKey = dh.pubkey
     var dh2 = new Diffie(strength, dh.prime, dh.generator, dh.privkey)
     var pubKey2 = dh2.GetPubKey()
     expect(pubKey).to.deep.equal(pubKey2)
 
   })
   describe('Get Pubkey', () => {
-    it('should generate a key', function() {
+    it('generates a key', function () {
```
Comment (tfredett) [inline test/Secure.js]: Note: capital `G`

File: test/Agent.js [hunk 4]
```diff
@@ -8,62 +8,61 @@ var expect = chai.expect
...
 
-    it('Should identify as identity when cast as identity', function(){
+    it('Identifies as an identity when cast as identity', function () {
         var agent = new Agent(UserLib.Identity)
         expect(agent.user.constructor.name).to.equal("Identity")
         expect(agent.user.type).to.equal(UserType.Identity)
     })
 
-    it('Should identify of type Unloq as identity when cast as identity of Unloq', function(){
+    it('Identifies as a Unloq identity when cast as a Unloq identity', function () {
```
Comment (dawnTestCode) [inline test/Agent.js]: "as an Unloq identity" ...

File: test/Agent.js [hunk 5]
```diff
@@ -8,62 +8,61 @@ var expect = chai.expect
...
-    it('Should identify as generic when not cast', function(){
+
+    it('Identifies as generic when not cast', function () {
         var agent = new Agent()
         expect(agent.user.constructor.name).to.equal("User")
         expect(agent.user.type).to.equal(UserType.Generic)
     })
 
-    it('Should allow server to generate random', function(){
+    it('Allows a server to generate random', function () {
```
Comment (dawnTestCode) [inline test/Agent.js]: While I know this was not in the original test, still ... "allow a server to generate random" .... what? Or is it generate x based on random y? Anyway, a bit more descriptive here will be good.

File: test/Agent.js [hunk 6]
```diff
@@ -8,62 +8,61 @@ var expect = chai.expect
...
         var agent = new Agent(UserLib.Server)
         var data = agent.user.Generate()
         expect(data).to.exist
     })
 
-    it('Should allow server to split random', function(){
+    it('Allows a server to split random', function () {
```
Comment (dawnTestCode) [inline test/Agent.js]: See line 42

File: test/Agent.js [hunk 7]
```diff
@@ -8,62 +8,61 @@ var expect = chai.expect
...
         var agent = new Agent(UserLib.Server)
         var data = agent.user.Split(2, 2, 256)
         expect(data).to.exist
     })
 
-    it('Should allow server to split random and reassemble', function(){
+    it('Allows a server to split random and reassemble', function () {
```
Comment (dawnTestCode) [inline test/Agent.js]: See line 42

File: test/Agent.js [hunk 8]
```diff
@@ -8,62 +8,61 @@ var expect = chai.expect
...
         var agent = new Agent(UserLib.Server)
         var key = agent.Generate()
         var shares = agent.Split(2, 2, 256)
         var combined = agent.Combine(shares.value)
-        expect(combined.value).to.equal(key.value)        
+        expect(combined.value).to.equal(key.value)
     })
 
-    it('Should not allow generic agent to generate random', function(){
+    it('Prevents a generic agent to generate random', function () {
```
Comment (dawnTestCode) [inline test/Agent.js]: See line 42

File: test/Agent.js [hunk 9]
```diff
@@ -8,62 +8,61 @@ var expect = chai.expect
...
         var agent = new Agent()
-        expect(function(){
+        expect(function () {
             agent.Generate()
         }).to.throw('Method not implemented.')
     })
-})
+})
```
Comment (dawnTestCode) [inline test/Agent.js]: Review for newlines. All files should end with a newline

File: test/Coval.js [hunk 10]
```diff
@@ -5,5 +5,5 @@ var should = chai.should()
 var CovalLib = require("../build/Coval")
 
 describe('Coval.js', () => {
-    it('should enforce https everywhere')
+    it('Enforces https everywhere')
 })
```
Comment (dawnTestCode) [inline test/Coval.js]: newline check

File: test/Dat.js [hunk 11]
```diff
@@ -8,7 +8,7 @@ var fs = require("fs")
 
 describe('Dat', () => {
     describe('Share', () => {
-        it('should resolve provided path correcty', function(done) {
+        it('Resolves a provided path correctly', function(done) {
```
Comment (dawnTestCode) [inline test/Dat.js]: Pop spaces before and after () after function

File: test/Emblem.js [hunk 12]
```diff
@@ -8,14 +8,14 @@ var Dat = require('../build/transport/Dat').Dat
 
 describe('Emblem', function(){
     describe('Add Dat', function(){
-        it('should sucessfully add dat to emblem', function(){
+        it('Successfully adds dat to emblem', function(){
```
Comment (dawnTestCode) [inline test/Emblem.js]: Space before and after parens in this file too

File: test/Emblem.js [hunk 13]
```diff
@@ -45,7 +45,7 @@ describe('Emblem', function(){
         })
     })
     describe('Claimed', ()=>{
-        it('should return false until claimed', ()=>{
+        it('Returns false until claimed', ()=>{
```
Comment (dawnTestCode) [inline test/Emblem.js]: rename this one to represent that claimed is false when a new emblem is crated

File: test/Lightrail.js [hunk 14]
```diff
@@ -7,7 +7,7 @@ var path = require("path")
 require("dotenv").config({path: path.join(__dirname, "..", "build", "test.env")})
 
 describe('Lightrail', () => {
-    it('should instantiate without options', ()=>{
+    it('Instantiates without options', ()=>{
```
Comment (dawnTestCode) [inline test/Lightrail.js]: Reword this one ... Instantiates successfully with default configuration

File: test/Lightrail.js [hunk 15]
```diff
@@ -18,7 +18,7 @@ describe('Lightrail', () => {
             additionalHeaders: {} })
     })
 
-    it('should instantiate with some options', () => {
+    it('Instantiates with some options', () => {
```
Comment (dawnTestCode) [inline test/Lightrail.js]: rename ..  Instantiates correctly with one option passed

File: test/Lightrail.js [hunk 16]
```diff
@@ -28,7 +28,7 @@ describe('Lightrail', () => {
             additionalHeaders: {} })
     })
 
-    it('should instantiate with some all options', () => {
+    it('Instantiates with all options', () => {
```
Comment (dawnTestCode) [inline test/Lightrail.js]: Instantiates correctly with all options passed

File: test/Lightrail.js [hunk 17]
```diff
@@ -44,7 +44,7 @@ describe('Lightrail', () => {
             additionalHeaders: {} })
     })
 
-    it('should allow creation of new contacts', ()=>{
+    it('Allows creation of new contacts', ()=>{
```
Comment (dawnTestCode) [inline test/Lightrail.js]: Allows creation of a new contact

File: test/Lightrail.js [hunk 18]
```diff
@@ -54,13 +54,13 @@ describe('Lightrail', () => {
             lastName: 'user' })
     })
 
-    it('should load with apiKey from environment', ()=>{
+    it('Loads with an apiKey from the environment', ()=>{
```
Comment (dawnTestCode) [inline test/Lightrail.js]: loads with an api key from an environment varible

File: test/Lightrail.js [hunk 19]
```diff
@@ -69,7 +69,7 @@ describe('Lightrail', () => {
             })            
         })
 
-        it('should handle lost connection gracefully')
+        it('Handles a lost connection gracefully')
     })
         
 })
```
Comment (dawnTestCode) [inline test/Lightrail.js]: newline

File: test/Lightrail.js [hunk 20]
```diff
@@ -54,13 +54,13 @@ describe('Lightrail', () => {
...
 
-    it('should load with apiKey from environment', ()=>{
+    it('Loads with an apiKey from the environment', ()=>{
         var Lightrail = new LightrailLib({apiKey: process.env.LIGHTRAIL_API_KEY}).client
         expect(Lightrail.configuration.apiKey).to.exist
     })
 
     describe('Create Contact', () => {
-        it('should create new user', function(done) {
+        it('Creates a new user', function(done) {
```
Comment (dawnTestCode) [inline test/Lightrail.js]: spaces before and after ()

File: test/Multichain.js [hunk 21]
```diff
@@ -47,7 +47,7 @@ describe('Multichain', () => {
                 })
             })
     
-            it('should return empty list when key not found', function(done) {
+            it('Returns an empty list when key not found', function(done) {
```
Comment (dawnTestCode) [inline test/Multichain.js]: returns an empty list with no error when key not found

File: test/Secure.js [hunk 22]
```diff
@@ -12,51 +12,51 @@ var Pre = Coval.Secure.Pre
...
   })
 
-  it('should return expected value', function () {
+  it('Returns expected value', function () {
     var _protected = new Protected(3)
     expect(_protected.Value().GetValue()).to.equal(3)
   })
 
-  it('should not allow value to be accessed twice', function () {
+  it('Prevents value to be accessed twice', function () {
```
Comment (dawnTestCode) [inline test/Secure.js]: prevents value access after initial access

File: test/Secure.js [hunk 23]
```diff
@@ -12,51 +12,51 @@ var Pre = Coval.Secure.Pre
...
     var _protected = new Protected("this msg will self destruct")
     expect(_protected.Value().GetValue()).to.equal("this msg will self destruct")
     expect(_protected.Value().GetValue()).to.equal(null)
   })
 
-  it('should record error when read attempted twice', function () {
+  it('Records an error on second read attempt', function () {
```
Comment (dawnTestCode) [inline test/Secure.js]: on second access attempt

File: test/Secure.js [hunk 24]
```diff
@@ -92,15 +92,15 @@ describe('Shamir', function () {
...
+    it('Splits key into 2 shares', function () {
       var serverObject = new Shamir.Key()
       var shares = serverObject.CreateShares(2, 2).GetValue()
       expect(shares).to.not.be.empty
       expect(shares).to.be.an('Array')
     })
   })
   describe('CombineShares', function () {
-    it('should combine shares', function () {
+    it('Combine shares', function () {
```
Comment (dawnTestCode) [inline test/Secure.js]: combines shares

File: test/envelope.js [hunk 25]
```diff
@@ -5,19 +5,19 @@ var Coval = require('../build/Coval')
 var Envelope = require('../build/transport/Envelope').Envelope
 describe('Envelope', function() {
     
-        it('should be able to assign value', function(){
+        it('Is able to assign value', function(){
```
Comment (dawnTestCode) [inline test/envelope.js]: Assigns a value

File: test/Agent.js (hunk 4 above)
Comment (tfredett) [reply test/Agent.js]: done

File: test/Agent.js (hunk 9 above)
Comment (tfredett) [reply test/Agent.js]: done

File: test/Coval.js (hunk 10 above)
Comment (tfredett) [reply test/Coval.js]: done

File: test/Dat.js (hunk 11 above)
Comment (tfredett) [reply test/Dat.js]: done

File: test/Emblem.js (hunk 12 above)
Comment (tfredett) [reply test/Emblem.js]: done

File: test/Emblem.js (hunk 13 above)
Comment (tfredett) [reply test/Emblem.js]: done

File: test/Lightrail.js (hunk 14 above)
Comment (tfredett) [reply test/Lightrail.js]: done

File: test/Lightrail.js (hunk 15 above)
Comment (tfredett) [reply test/Lightrail.js]: done

File: test/Lightrail.js (hunk 16 above)
Comment (tfredett) [reply test/Lightrail.js]: done

File: test/Lightrail.js (hunk 17 above)
Comment (tfredett) [reply test/Lightrail.js]: done

File: test/Lightrail.js (hunk 18 above)
Comment (tfredett) [reply test/Lightrail.js]: done

File: test/Lightrail.js (hunk 20 above)
Comment (tfredett) [reply test/Lightrail.js]: done

File: test/Lightrail.js (hunk 19 above)
Comment (tfredett) [reply test/Lightrail.js]: done

File: test/Multichain.js (hunk 21 above)
Comment (tfredett) [reply test/Multichain.js]: done

File: test/Secure.js (hunk 22 above)
Comment (tfredett) [reply test/Secure.js]: done

File: test/Secure.js (hunk 23 above)
Comment (tfredett) [reply test/Secure.js]: done

File: test/Secure.js (hunk 24 above)
Comment (tfredett) [reply test/Secure.js]: done

File: test/envelope.js (hunk 25 above)
Comment (tfredett) [reply test/envelope.js]: done

File: test/Agent.js (hunk 5 above)
Comment (tfredett) [reply test/Agent.js]: done, Talked with @genecyber, and it looks like it is generating an Envelope object. Updated the test title as such.

File: test/Agent.js (hunk 6 above)
Comment (tfredett) [reply test/Agent.js]: done

File: test/Agent.js (hunk 7 above)
Comment (tfredett) [reply test/Agent.js]: done

File: test/Agent.js (hunk 8 above)
Comment (tfredett) [reply test/Agent.js]: done

**Comment by dawnTestCode** [comment]
I added a few more comments ....

File: test/Agent.js [hunk 26]
```diff
@@ -39,27 +39,27 @@ describe('Agent', () => {
         expect(agent.user.type).to.equal(UserType.Generic)
     })
 
-    it('allows a server to generate random', function () {
+    it('allows a server to generate a random envelope', function () {
```
[the remark this answers is not in the document]
Comment (tfredett) [reply test/Agent.js]: good call, that makes more sense. updated.

File: test/Agent.js [hunk 27]
```diff
@@ -39,27 +39,27 @@ describe('Agent', () => {
...
         var agent = new Agent(UserLib.Server)
         var data = agent.user.Generate()
         expect(data).to.exist
     })
 
-    it('allows a server to split random', function () {
+    it('allows a server to split a random envelope', function () {
```
[the remark this answers is not in the document]
Comment (tfredett) [reply test/Agent.js]: good call, that makes more sense. updated.

File: test/Agent.js [hunk 28]
```diff
@@ -39,27 +39,27 @@ describe('Agent', () => {
...
         var agent = new Agent(UserLib.Server)
         var data = agent.user.Split(2, 2, 256)
         expect(data).to.exist
     })
 
-    it('allows a server to split random and reassemble', function () {
+    it('allows a server to split and reassemble a random envelope', function () {
```
[the remark this answers is not in the document]
Comment (tfredett) [reply test/Agent.js]: good call, that makes more sense. updated.

File: test/Agent.js [hunk 29]
```diff
@@ -39,27 +39,27 @@ describe('Agent', () => {
...
         var agent = new Agent(UserLib.Server)
         var key = agent.Generate()
         var shares = agent.Split(2, 2, 256)
         var combined = agent.Combine(shares.value)
         expect(combined.value).to.equal(key.value)
     })
 
-    it('prevents a generic agent to generate random', function () {
+    it('prevents a generic agent from generating a random envelope', function () {
```
[the remark this answers is not in the document]
Comment (tfredett) [reply test/Agent.js]: good call, that makes more sense. updated.

**Pull request merged**